x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/routes.tsx

```diff
@@ -25,6 +26,7 @@ const CloudPostureSpyRoute = ({ pageName, ...rest }: { pageName?: CloudSecurityP
 const cspSecuritySolutionContext: CspSecuritySolutionContext = {
   getFiltersGlobalComponent: () => FiltersGlobal,
   getSpyRouteComponent: () => CloudPostureSpyRoute,
+  useExpandableFlyoutApi,
```

Comment:
It is mind-bending that we can't import directly into our plugin and use it, it only works if all this wrapping magic happened. but glad that it works for us

Reply:
yeah exactly, that's why i wanted to try and explain it. its really important to understand if we'd ever make future decisions based on this option